releases/25.05.0/25.05.0.md

```diff
@@ -0,0 +1,256 @@
+  * Fixed graph deactivation error “Failed to get user data from graph. Constructed with invalid values”  
+* Refactoring the MovieSystem so that it may be moved into a gem  
+* Asset Processor analysis regarding how many assets are new files, outdated, have altered dependencies, or have new builders.
+* Added filter to prevent the use of reserved words in the project names (e.g. "project", "name", "o3de")
+
+# Sig-Simulation
+
+* SimulationInterfaces Gem: new Gem implementing standard ROS 2 simulation interfaces  
+* ROS2 Gem: added segmentation feature to LiDAR sensor  
+* ROS2 Gem: added notification bus to the spawner  
+* ROS2 Gem: reimplemented PID controller  
+* ROS2 Gem: fixed multiple bugs in ROS2Clock and joint manipulator; removed not working Python tests  
+* WarehouseAssets assets-only Gem: meshes and prefabs were optimized  
+* LevelGeoreferencing Gem: new Gem was created based on functionality from ROS 2 Gem  
+* ProteusRobot assets-only Gem: the Gem was removed, the content was moved to ROS2RobotSamples Gem  
+* RosRobotSample assets-only Gem: the Gem was removed, the content was moved to ROS2RobotSamples Gem  
+* WarehouseSample assets-only Gem: the Gem was removed  
+* Ros2FleetRobotTemplate: the template was simplified (removed render passes, removed one level)  
+* Ros2ProjectTemplate: the template was regenerated (new level based on WarehouseAssets instead of WarehouseSample)  
+* Ros2RoboticManipulationTemplate: the template was simplified (removed one level, render passes and many assets)  
+* Ros2RoboticManipulationTemplate: assets for Panda Franka robot were moved to ROS2RobotSamples Gem
+
+
+  
+# Sig-Core
+
+* RAM reduction by removing legacy asset table  
+* Resolved issues with latest stable version of CMake
+
+
+  
+# Sig-Networking
+
+* Fix player spawning when client connects to a server during a server level load  
+* AWS cloud service gems moved to separate repo to keep engine agnostic. See [aws/o3de-repo](https://github.com/aws/o3de-repo)  
+* Non-monolithic release mode servers supported  
+* Fix spawning network prefabs containing more than one entity  
+* Multiplayer component autogen throws compile time error if accidentally creating multiplayer components with the same name  
+* Game session “level” property added to automatically loading levels when a multiplayer game session begins  
+* Better checks and warnings around invalid IP addresses
+
+# Sig-Platform
+
+* Addressed the Python shared library conflict on Linux, which caused segmentation faults when using the ROS2 environment and Ros2 GEM with O3DE.  
+* Disabled Android's APK compression, allowing PAK files to load without significant delays or softlocks.  
+* Fixed Python script execution errors in the Shader Management Console on Linux, allowing shader variants to be created successfully.  
+* Proposed an RFC to add curated and uncurated lists of remote repositories to the project manager, improving discoverability.  
```

Comment:
Hyperlink to the rfc if any

Reply:
Actually I will delete that line as it doesn't affect the release at all.